gocourse04/zoo/sector_test.go

```diff
@@ -0,0 +1,37 @@
+package zoo
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewSector(t *testing.T) {
+	sector := NewSector("Test sector")
+
+	assert.Equal(t, "Test sector", sector.Subtype)
+	assert.Equal(t, 0, len(sector.Animals))
+}
+
+func TestSectorAddAnimal(t *testing.T) {
+	sector := *NewSector("Test sector")
```

Comment:
Можемо більш реалістичні назви придумати?

Наприклад, 
```suggestion
	sector := *NewSector("Sector A")
```